src/Options/docs.js

```diff
@@ -82,6 +82,7 @@
 
 /**
  * @interface PagesOptions
+ * @property {Generic[]} customRoutes The custom routes.
```

Comment:
Generic is a type?

Reply:
Uh, yes, that was auto-gen, it should be `PagesRoute[]`, see the definition in index.js:
```
/* The custom routes.
  :DEFAULT: [] */
  customRoutes: ?(PagesRoute[]);
```

I think the definitions builder does not support array of custom types, looking into this...

Reply:
Fixed it, so as a side effect of this PR, the definitions builder now also supports array of custom type.